learning_journal/templates/new-entry.html

```diff
@@ -6,16 +6,16 @@
 <main id="content" class="container">
   <a href="/" type="button" class="btn btn-info btn-arrow-left">Home</a>
   <section id="journal-entry-form">
-    <form>
+    <form action="{{ request.route_url('home') }}" method="POST">
```

Comment:
Is your 'home' route set up to handle newly submitted journal entries?  This seems a bit odd.